src/main/java/it/unisa/Amigo/gruppo/controller/GruppoController.java

```diff
@@ -97,11 +138,127 @@ public String removeMembro(@PathVariable(name = "idPersona") int idPersona,@Path
+     * @return il path della pagina su cui eseguire il redirect
+     */
+    @GetMapping("/gruppi/{id}/commissione_detail/{id_commissione}")
+    public String findAllMembriInCommissione(Model model, @PathVariable(name = "id")int idSupergruppo, @PathVariable(name = "id_commissione")int idCommissione){
+        Persona personaLoggata = gruppoService.getAuthenticatedUser();
+        prepareCandidateList(idCommissione,model,gruppoService.findAllMembriInSupergruppo(idCommissione));
+        model.addAttribute("isCapogruppo", gruppoService.isResponsabile(personaLoggata.getId(), gruppoService.findGruppoByCommissione(idCommissione).getId()));
+        model.addAttribute("isResponsabile", gruppoService.isResponsabile(personaLoggata.getId(),idCommissione));
+        return "gruppo/commissione_detail";
+    }
+
+    /**
+     * Esegue la chiusura di una commissione @{@link Commissione}
+     * @param model per salvare informazioni da recuperare nell'html
+     * @param idCommissione l'id della commissione della quale si vuole effettuare la chiusura
+     * @return il path della pagina su cui eseguire il redirect
+     */
+    @GetMapping("/gruppi/commissioni/{id2}/chiusura")
+    public String closeCommissione(Model model, @PathVariable(name = "id2")int idCommissione){
+        Persona personaLoggata = gruppoService.getAuthenticatedUser();
+        prepareCandidateList(idCommissione,model,gruppoService.findAllMembriInSupergruppo(idCommissione));
+        model.addAttribute("isCapogruppo", gruppoService.isResponsabile(personaLoggata.getId(), gruppoService.findGruppoByCommissione(idCommissione).getId()));
+        model.addAttribute("isResponsabile", gruppoService.isResponsabile(personaLoggata.getId(),idCommissione));
+        model.addAttribute("flagChiusura",1);
+        gruppoService.closeCommissione(idCommissione);
+        return "gruppo/commissione_detail";
+    }
+
+    /**
+     * Gestisce il form necessario alla creazione di una nuova commissione @{@link Commissione}
+     * @param model per salvare informazioni da recuperare nell'html
+     * @param idSupergruppo l'id del gruppo al quale verrà assegnata la nuova commissione
+     * @return il path della pagina su cui eseguire il redirect
+     */
+    @GetMapping("/gruppi/{id}/commissioni/create")
+    public String createCommissioneForm(Model model, @PathVariable(name = "id")int idSupergruppo){
+        model.addAttribute("idGruppo", idSupergruppo);
+        model.addAttribute("command", new GruppoFormCommand());
+        return "gruppo/crea_commissione";
+    }
+
+    /**
+     * Gestisce la creazione di una commissione @{@link Commissione}
+     * @param gruppoFormCommand il gestore del form
+     * @param model per salvare informazioni da recuperare nell'html
+     * @param idGruppo l'id del gruppo al quale verrà assegnata la nuova commissione
+     * @return il path della pagina su cui eseguire il redirect
+     */
+    @PostMapping("/gruppi/{idGruppo}/commissioni/createCommissione")
```

Comment:
```suggestion
    @PostMapping("/gruppi/{idGruppo}/commissioni/create")
```
Si può usere lo stesso url di quello che mostra il form, perché sono mapati su due metodi HTTP diversi.